src/legacy/core_plugins/vis_type_metric/public/types.ts

```diff
@@ -16,16 +16,34 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { CoreStart, Plugin } from 'kibana/public';
-import { initMetricVisLegacyModule } from './metric_vis_legacy_module';
 
-export class LegacyDependenciesPlugin implements Plugin<void, void> {
-  public setup() {
-    // Init kibana/metric_vis AngularJS module.
-    initMetricVisLegacyModule();
-  }
+import { ColorSchemas } from 'ui/vislib/components/color/colormaps';
+import { RangeValues } from 'ui/vis/editors/default/controls/ranges';
+import { ColorModes } from '../../kbn_vislib_vis_types/public/utils/collections';
+import { Labels, Style } from '../../kbn_vislib_vis_types/public/types';
 
-  public start(core: CoreStart) {
-    // nothing to do here yet
-  }
+export const visType = 'metric';
+
+export interface DimensionsVisParam {
+  metrics: any;
```

Comment:
There is an interface that describes a dimension object https://github.com/elastic/kibana/blob/master/src/legacy/ui/public/visualize/loader/pipeline_helpers/build_pipeline.ts#L34. Let's use it instead of `any`.

Reply:
Done.